docs/cloud/cloud-troubleshooting/issue-cloud-version-unknown.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Database "Unknown"
+layout: default
+parent: Cloud troubleshooting
+grand_parent: Cloud
+nav_order: 4
+---
+
+# Why is my database's version "Unknown"?
+
+| Database Status | Description | 
+|---|---|
+| CREATING/PROVISIONING | The database does not fully exist yet, and as such the database's version is unknowable. |
+| DEPROVISIONING/DELETED |  If a database is deleted before it finishes creating/provisioning, then it may not have a version | 
+| FAILED | If a database fails to start properly then it may not have a version, or may not respond properly to version requests |
```

Comment:
There are more database statuses than, I've added the ones I know we are going to need and written a blurb about that they mean here: https://github.com/molecula/m-cloud-control-plane/pull/618/files#diff-de2c7c520bd1e539f5e903473374db08d059b837a18384382e62a803a1c28b93R308